includes/sanitizers/class-amp-link-sanitizer.php

```diff
@@ -31,14 +31,32 @@ class AMP_Link_Sanitizer extends AMP_Base_Sanitizer {
 	 */
 	const DEFAULT_META_CONTENT = 'AMP-Redirect-To; AMP.navigateTo';
 
+	/**
+	 * The rel attribute value for AMP links.
+	 *
+	 * @var string
+	 */
+	const REL_VALUE_AMP = 'amphtml';
+
+	/**
+	 * The rel attribute value that will force non-AMP links.
+	 *
+	 * Normally, in a paired mode, links to the same origin will be for AMP.
+	 * But by adding this rel value, the link will be to non-AMP.
+	 *
+	 * @var string
+	 */
+	const REL_VALUE_NON_AMP_TO_AMP = 'not-amphtml';
```

Comment:
I don't necessarily have a better suggestion than `not-amphtml` but I don't feel entirely satisfied with this.

Maybe instead of `not-amphtml` it should be just `noamphtml`. This could make it more inline with `nofollow`, noopener`, and `noreferrer`. Note there is a [`noopener`](https://html.spec.whatwg.org/multipage/links.html#link-type-noopener) _and_ an [`opener`](https://html.spec.whatwg.org/multipage/links.html#link-type-opener) link relation, as seen in the [Link Relations](https://www.iana.org/assignments/link-relations/link-relations.xhtml) registry:

> ![image](https://user-images.githubusercontent.com/134745/72845573-c0092b80-3c53-11ea-924a-2605848ef7ea.png)


So having an `amphtml` and a `noamphtml` would seem like it better follows this precedent.

```suggestion
	const REL_VALUE_NON_AMP_TO_AMP = 'noamphtml';
```

This would also help avoid confusion between `not-amphtml` and `non-amphtml`.

Reply:
Good point, from that [Link Relations link](https://www.iana.org/assignments/link-relations/link-relations.xhtml) you shared, `no*` is the convention.

Reply:
Committed with https://github.com/ampproject/amp-wp/pull/4146/commits/640834ff7046455cbe91138e388c8e9f6e8147be